packages/client/src/controllers/crypto.ts

```diff
@@ -30,6 +35,10 @@ export class KeyChain implements IKeyChain {
     return getLoggerContext(this.logger);
   }
 
+  get storageKey(): string {
+    return this.client.storagePrefix + this.version + "//" + formatStorageKeyName(this.context);
```

Comment:
This can probably be abstracted into a reusable util function, I wanted to keep things simple during refactoring and not also have to symlink in `@walletconnect/utils`